migrations/20201024130703_stage_info_test_data.mysql.up.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO records VALUES (9000, 3, 1, TRUE, 0, 1, 100.02);
+INSERT INTO records VALUES (9001, 4, 1, TRUE, 0, 1, 200.11);
+INSERT INTO records VALUES (9002, 5, 1, TRUE, 0, 1, 150.25);
+INSERT INTO records VALUES (9003, 6, 1, TRUE, 0, 1, 300.45);
+INSERT INTO records VALUES (9004, 3, 2, TRUE, 0, 1, 50.02);
+INSERT INTO records VALUES (9005, 4, 2, TRUE, 0, 1, 133.11);
+INSERT INTO records VALUES (9006, 5, 2, TRUE, 0, 1, 122.25);
+INSERT INTO records VALUES (9007, 6, 2, TRUE, 0, 1, 331.45);
```

Comment:
Insert ってまとめて出来なかったでしたっけ？

Reply:
```suggestion
INSERT INTO records VALUES (9000, 1, 1, TRUE, 0, 1, 100.02);
INSERT INTO records VALUES (9001, 1, 1, TRUE, 0, 1, 200.11);
INSERT INTO records VALUES (9002, 1, 1, TRUE, 0, 1, 150.25);
INSERT INTO records VALUES (9003, 1, 1, TRUE, 0, 1, 300.45);
INSERT INTO records VALUES (9004, 1, 2, TRUE, 0, 1, 50.02);
INSERT INTO records VALUES (9005, 1, 2, TRUE, 0, 1, 133.11);
INSERT INTO records VALUES (9006, 1, 2, TRUE, 0, 1, 122.25);
INSERT INTO records VALUES (9007, 1, 2, TRUE, 0, 1, 331.45);
```

後，第二要素の `player_id` は `FOREIGN KEY (player_id) REFERENCES player(id) ON DELETE CASCADE` としているので，該当する ID の Player が居ないと Migration に失敗します．現状以下の Insert しか Migration ファイルにはないので， `player_id` は 1 にしないと普通は死にます

```
INSERT INTO players VALUES (1, 'test', 1, '2020-05-02 12:48:35', '2020-05-02 12:48:35');
```

Reply:
なるほど、僕多分ローカルで自分でインサートしたことあるから player_id 3~6はあった。
プレイヤーIDとステージIDが同じのレコードが複数存在するのはおかしいので、プレイヤーをインサートするマイグレーションファイルを作ります